cmd/eksctl-anywhere/cmd/listpackages.go

```diff
@@ -21,13 +21,17 @@ var lpo = &listPackagesOption{}
 
 func init() {
 	listCmd.AddCommand(listPackagesCommand)
-	listPackagesCommand.Flags().Var(&lpo.source, "source", "Discovery Location. Options (cluster, registry)")
-	err := listPackagesCommand.MarkFlagRequired("source")
-	if err != nil {
-		log.Fatalf("Error marking flag as required: %v", err)
+
+	listPackagesCommand.Flags().Var(&lpo.source, "source",
+		"Packages info discovery source. Options: cluster, registry.")
+	listPackagesCommand.Flags().StringVar(&lpo.kubeVersion, "kube-version", "",
+		"Kubernetes version of the packages to list, for example: \"1.23\".")
```

Comment:
I think <major>.<minor> might be easier for the user?

Reply:
I don't understand what you mean here, can you elaborate?

Reply:
Ah, I see, you've edited your comment, and now I understand what you mean at least.

Reply:
Maybe he means?
```suggestion
		"Kubernetes version <major>.<minor> of the packages to list, for example: \"1.23\".")
```